contrib/hf-samples/2-node-network.groovy

```diff
@@ -0,0 +1,39 @@
+//! Simulation
+
+import org.arl.fjage.RealTimePlatform
+import org.arl.unet.sim.*
+import org.arl.unet.sim.channels.*
+
+///////////////////////////////////////////////////////////////////////////////
+// display documentation
+
+println '''
+2-node network (High-frequency)
+-------------------------------
+
+Node A: tcp://localhost:1101, http://localhost:8081/
+Node B: tcp://localhost:1102, http://localhost:8082/
+'''
+
+///////////////////////////////////////////////////////////////////////////////
+// simulation settings
+
+platform = RealTimePlatform           // use real-time mode
+
+///////////////////////////////////////////////////////////////////////////////
+// channel and modem settings
+
+channel = [
+  model:                BasicAcousticChannel,
+  carrierFrequency:     60.kHz,
+  bandwidth:            20000.Hz,
+  waterDepth:           30.m
+]
+
+modem.dataRate =        [10000.bps, 15000.bps]
```

Comment:
dataRate : 20 - 30 kbps?

Reply:
Since bandwidth was 20 kHz and since we never use the full bandwidth, I went with these numbers. This is for discussion. Need to decide on these.